Sources/Purchasing/Purchases/Purchases.swift

```diff
@@ -40,6 +40,11 @@ public typealias PurchaseCompletedBlock = @MainActor @Sendable (StoreTransaction
                                                                 PublicError?,
                                                                 Bool) -> Void
 
+/**
+ Completion block for ``Purchases/getStorefrontCountryCode(``
+ */
+public typealias GetStorefrontBlock = @MainActor @Sendable (Storefront?) -> Void
```

Comment:
This is not a big deal and probably better to leave like this for consistency with other similar types, but "block" is how the callback concept is named in Objective-C while "closure" is the Swift counterpart.
Usually, these are called `Completion`, so `GetStorefrontCompletion` would probably be a more iOS-y name.

Reply:
That's a good point... But yeah, I mostly wanted to be consistent, which I think it's better for now. Happy to deprecate these typealiases in favor of new ones in the future though!

Reply:
Yeah, we can think of better namings for a next major / API redesign. Besides, the closure typealias names are the least important, IMO